packages/bigtable/src/row.js

```diff
@@ -158,25 +123,58 @@ util.inherits(Row, common.GrpcServiceObject);
  * // }
  */
 Row.formatChunks_ = function(chunks) {
-  var families = [];
-  var chunkList = [];
+  var rows = [];
+  var familyName;
+  var qualifierName;
+
+  chunks.reduce(function(row, chunk) {
+    var family;
+    var qualifier;
+
+    row.data = row.data || {};
 
-  chunks.forEach(function(chunk) {
-    if (chunk.resetRow) {
-      chunkList = [];
+    if (chunk.rowKey) {
+      row.key = Mutation.convertFromBytes(chunk.rowKey);
     }
 
-    if (chunk.rowContents) {
-      chunkList.push(chunk.rowContents);
+    if (chunk.familyName) {
+      familyName = chunk.familyName.value;
+    }
+
+    if (familyName) {
+      family = row.data[familyName] = row.data[familyName] || {};
```

Comment:
Just looking at this line of code though, it seems like extend would do the same thing:

```js
var assert = require('assert')
var extend = require('extend')

var familyName = 'familyName'

// Test 1:
var row = {
  data: {
    familyName: null
  }
}

var familyDoubleAssignment = row.data[familyName] = row.data[familyName] || {}
var familyExtended = extend(row.data[familyName], {})
assert.deepEqual(familyDoubleAssignment, familyExtended)

// Test 2:
var row = {
  data: {
    familyName: {
      insideFamily: true
    }
  }
}

var familyDoubleAssignment = row.data[familyName] = row.data[familyName] || {}
var familyExtended = extend(row.data[familyName], {})
assert.deepEqual(familyDoubleAssignment, familyExtended)
```

Am I missing something easy?